backend/src/packages/orders/order.entity.ts

```diff
@@ -22,12 +28,16 @@ class OrderEntity implements IEntity {
 
   private businessId: OrderEntityT['businessId'];
 
-  private driverId: OrderEntityT['driverId'];
-
   private customerName: OrderEntityT['customerName'] | null;
 
   private customerPhone: OrderEntityT['customerPhone'] | null;
 
+  private shiftId: ShiftEntity['id'];
+
+  private driver: DriverInfo | null;
+
```

Comment:
why we add driver to the entity if it used only for responses? Entity should correspond to database entity as I see it, the same for the truck field.  If we create new order there is no driver and truck field in it. We can add driver to resposeDto and use it for typing